client/stylesheets/_toCat.scss

```diff
@@ -4597,12 +4605,11 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .searchResultsCardLeft {
-    width: 30% !important;  
-    height: 200px;
+    width: 30% !important;
 }
 
 .searchResultsCardRight {
-    width: 40%;
+    width: 60%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-left, vertical-align, width